lib/common/grpc-service.js

```diff
@@ -332,24 +364,20 @@ GrpcService.convertValue_ = function(value) {
     convertedValue = {
       structValue: GrpcService.objToStruct_(value)
     };
-  } else if (is.date(value)) {
```

Comment:
Thanks all, and sorry again for dragging this out. So long as it's not possible for Google to publish a new proto type that's essentially the same as Struct but with one of the permissible kinds being Timestamp, then this change makes sense. Glad to hear that there are plans to make it possible to query timestamps regardless.

(It actually does seem possible to use the protoPayload field, with `@type: 'type.googleapis.com/google.protobuf.Struct'`... at least it works from the API explorer -- but I see no difference between that and jsonPayload/structPayload.)